### PR TITLE
updated for psd2 and recurring

### PR DESCRIPTION
**Pull request by alfmags**

### CODE REVIEW

File: vipps-psp-api.md [hunk 1]
```diff
@@ -204,6 +204,123 @@ Note 93 is for when the makePayment request from Vipps contains the statuses CAN
...
+
+## Subscription and oneclick scopes
+
+As of now, there are two possible ways to perform a recurring payment - *subscription* and *oneclick*. These are referred to as the *scope* of the recurring agreement. Only one *scope* can be used at a time, and it's not possible to change the scope of an agreement.
+
+*Subscription* based payments are created as a consent to an agreement that allows the PSP to withdraw money on unknown intervals. This implies that the user won't have to accept the payment on each occasion, only the first one when consenting to the agreement. An example of this could be a subscription to a music streaming service.
+
+*Oneclick* payments are created as agreements where the user will be prompted on each payment. An example of a oneclick payment might be a ticketing service where the user will get prompted to buy a new ticket when the old one expires. Instead of switching to the app to accept the payment, the user already has a oneclick recurring agreement with Vipps as the payment source, and the ticketing app won´t need to involve the Vipps app at all.
```
Comment (Larsera) [inline vipps-psp-api.md]: Do we still have oneclick in this form?

File: vipps-psp-api.md [hunk 2]
```diff
@@ -204,6 +204,123 @@ Note 93 is for when the makePayment request from Vipps contains the statuses CAN
...
+}
+```
+
+In the same way as a normal, non-recurring PSPv2 payment, the PSP will receive a *makePayment* callback. In the body of this callback, you will now also find a *userToken*.
+
+## The userToken
+
+The user token is a token generated when the user has given a consent. This token is provided to the PSP in the makePayment callback when initializing or preforming a recurring payment.
+
+A *userToken* contains the *scope* of the consent, in the claim named *scope*. The token also includes various useful information
```
Comment (Larsera) [inline vipps-psp-api.md]: Is it still called "scope"? I thought I heard that we are changing that…

Comment (alfmags) [reply vipps-psp-api.md]: Yes still called scope.

File: vipps-psp-api.md (hunk 1 above)
Comment (alfmags) [reply vipps-psp-api.md]: Good catch, we are not launching with oneclick

File: vipps-psp-api.md [hunk 3]
```diff
@@ -133,6 +133,123 @@ Note 93 is for when the makePayment request from Vipps contains the statuses CAN
 
 To provide a consistent end user experience it is important that Vipps is notified by changes to the payment status when it is captured, cancelled or refunded. Vipps also provides an endpoint to check the payment status.
 
+# PSD2 Compliance
+
+In order to be allow for delegated SCA through the PSD2 directive every transaction needs to be marked with Vipps's WalletId. Vipps's WalletId is A62.
```
Comment (Larsera) [inline vipps-psp-api.md]: Is this WalletId not specific for any card issuer? And would they know how to mark the transaction?

**Pull request merged**